samples/ArtistSite/ArtistSite.App/Components/ArtworkPanel.razor.cs

```diff
@@ -13,7 +13,7 @@ public string IconURL
         {
             get
             {
-                return $"Images/{Artwork.IconUri}";
+                return $"artwork/{Artwork.IconUri}";
```

Comment:
The directory below the base image address should be configurable rather than hard-coded.